### ISSUE TITLE
Implement (normalized) mutual infomation score

### ISSUE THREAD

**Issue by TomAugspurger** [dask/dask-ml#183]

**Comment by kshitij12345** [comment]
@TomAugspurger Can I give this a shot?

**Comment by TomAugspurger** [comment]
Yes, please do! Let me know if you have any trouble getting started. http://ml.dask.org/contributing.html has the contributing instructions.

**Comment by LuisMoralesAlonso** [comment]
@kshitij12345 Are you actually working on this?

**Comment by kshitij12345** [comment]
@LuisMoralesAlonso  actually was a bit busy, was planning to work on it this weekend, will inform if I am unable to

**Comment by LuisMoralesAlonso** [comment]
thanks!!!!
